ocaml/libs/xapi-stdext/lib/xapi-stdext-unix/unixext.mli

```diff
@@ -120,6 +120,32 @@ val open_connection_fd : string -> int -> Unix.file_descr
 
 val open_connection_unix_fd : string -> Unix.file_descr
 
+module CBuf : sig
+  type t = {
```

Comment:
Please add the doccoments from the deleted code, it's not obvious that cbuf stands for circular buffer.